capstone-frontend/src/components/CommentCard.jsx

```diff
@@ -0,0 +1,39 @@
+import React, { Component } from 'react';
+
+import '../styles/Groups.css';
+
+
+class CommentCard extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      user: {}
+    }
+  }
+
+  fetchUserData = async () => {
+    await fetch(`/api/user?id=${this.props.comment.userID}`)
+        .then(response => response.json()).then(response => this.setState({user: response}))
```

Comment:
Like Antonio's comment earlier, consider renaming the second `response` variable to something like `userJson`.

Reply:
Good idea, forgot to change it everywhere. Will update.